.github/workflows/release.yml

```diff
@@ -234,6 +240,42 @@ jobs:
           disable-run-tests: ${{ env.DISABLE_RUN_TESTS }}
           artifacts-dir: ${{ matrix.artifacts-dir-prefix }}-${{ needs.allocate-runners.outputs.version }}
 
+  build-windows-artifacts:
+    name: Build Windows artifacts
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ${{ needs.allocate-runners.outputs.windows-runner }}
+            arch: x86_64-pc-windows-msvc
+            features: servers/dashboard
+            artifacts-dir-prefix: greptime-windows-x86_64
+          - os: ${{ needs.allocate-runners.outputs.windows-runner }}
+            arch: x86_64-pc-windows-msvc
```

Comment:
We can keep consistent with other artifacts, for example, `greptime-windows-amd64-*`. It will help users to write download scripts, and no need to handle different styles.